src/CMakeLists.txt

```diff
@@ -199,6 +201,34 @@ INSTALL (FILES ${PROJECT_BINARY_DIR}/${BINARY_NAME}.timer
             DESTINATION usr/lib/systemd/system)
 
 
+###### I18N FILES ######
+
+SET(_locales
+    zh_CN
+    )
+
+IF(_locales)
+    FILE(MAKE_DIRECTORY "${PROJECT_BINARY_DIR}/i18n")
+
+    FOREACH(locale ${_locales})
+        SET(_po "${PROJECT_SOURCE_DIR}/i18n/${locale}.po")
+        SET(_mo "${PROJECT_BINARY_DIR}/i18n/${locale}.mo")
+        ADD_CUSTOM_COMMAND(
+            OUTPUT ${_mo}
+            COMMAND msgfmt -o ${_mo} ${_po}
```

Comment:
msgfmt should be found with FIND_PROGRAM() actually. this would also be the better condition for this whole IF block.